XmlFormatter/src/Windows/MainForm.cs

```diff
@@ -368,7 +400,7 @@ private void MainForm_DragEnter(object sender, DragEventArgs e)
             }
             string fileName = files[0];
             FileInfo fi = new FileInfo(fileName);
-            if (e.Data.GetDataPresent(DataFormats.FileDrop) && fi.Extension.ToLower() == ".xml")
+            if (e.Data.GetDataPresent(DataFormats.FileDrop) && fi.Extension.ToLower() == "." + formatterToUse.Extension)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5494671)